.github/workflows/kind-e2e-upgrade.yaml

```diff
@@ -116,6 +121,13 @@ jobs:
         # Create a cluster!
         kind create cluster --config kind.yaml
 
+    - name: Deploy certificates for upstream traffic
+      if: matrix.upstream-traffic == 'tls'
+      run: |
```

Comment:
Maybe add `set -x` here too as above?

Reply:
I think let's leave it as it is. The action file have no consistent but `set -x` is not so much useful.